sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -74,6 +88,17 @@
 })
 public class BigQueryUtils {
 
+  public static Descriptor getDescriptorFromTableSchema(TableSchema jsonSchema)
```

Comment:
How will the Descriptor be used ?

Reply:
Added comment. Vortex takes protocol buffers as elements, and the protocol buffer descriptor must match the table schema